recipes-extended/iptables/files/iptables.rules

```diff
@@ -0,0 +1,3 @@
+*filter
```

Comment:
Please add more details to commit message: state that we set default forward to drop.

Reply:
Done